### DISCUSSION TITLE
GitHub Mobile is blocking third-party OAuth on Android

### DISCUSSION THREAD

**Discussion by MuTsunTsai** [community/community#12960] [Mobile]
When GitHub Mobile app is set as the default app for GitHub web addresses, GitHub OAuth won't work with third-party websites in the browser (since OAuth redirection is intercepted by the app, which then ignores it). Please fix this bug as it is really quite annoying that I have to temporarily disable the app to use the OAuth.

**Comment by stevepopovich** [comment]
Hi Mu-Tsun, this sounds super annoying, thanks for bringing it to our attention. Do you have an example site that uses GitHub OAuth that you are having problems with? Also, do you have your steps for how you disable the interception to workaround this?

**Reply by MuTsunTsai** [reply]
1. GitLab is one example. 
2. Long press the GitHub app icon, click “app info” on the menu, go to “set as default” section, on “go to supported addresses” choose “use other apps” instead.

**Reply by stevepopovich** [reply]
Hi again Mu-Tsun, we believe that we will have a fix for this in the next release. I would like to invite you to our beta tester program so you can get the fix a week earlier. Please follow [this link](https://play.google.com/apps/testing/com.github.android) to join our beta program and get the fix in the next release.

**Reply by MuTsunTsai** [reply]
Good to know. I've been in beta for quite a while already.
